openmetadata-ui/src/main/resources/ui/src/components/Task/TaskTab/task-tab.less

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+.assignees-edit-input {
+  .ant-space-item:first-child {
+    width: 100%;
+  }
+}
```

Comment:
I feel it's not required, Remove use of space so that we don't need this CSS

Reply:
We have a common component inline edit and space is being used there so we will need this.